modules/mobile/app/controllers/mobile/v0/community_care_providers_controller.rb

```diff
@@ -52,7 +52,8 @@ def coordinates
       end
 
       def facility_coordinates
-        facility = Mobile::FacilitiesHelper.get_facilities(Array(params[:facilityId])).first
+        v1_facilities_flag = Flipper.enabled?(:mobile_v1_lighthouse_facilities, @current_user)
```

Comment:
not ideal. it'd be cleaner to have the user instance variable in the facility helper but since it's all module functions, instance variables won't work. This will be removed once we test the v1 LH facilities so better not to rework it or something that's going to be taken out.